codesign/lib/src/google_cloud_storage.dart

```diff
@@ -14,19 +14,17 @@ class GoogleCloudStorage {
   GoogleCloudStorage({
     required this.processManager,
     required this.rootDirectory,
-    required this.gCloudDownloadUploadPath,
   });
 
   final ProcessManager processManager;
   final Directory rootDirectory;
-  final String gCloudDownloadUploadPath;
   final String bucketPrefix = 'gs://flutter_infra_release';
 
   /// Method to upload code signed flutter engine artifact to google cloud bucket.
   Future<void> uploadEngineArtifact({
     required String from,
+    required String gCloudUploadPath,
```

Comment:
Since this is already in a GCS service class, you can simplify it back to `destination`

Reply:
Sure, renamed.